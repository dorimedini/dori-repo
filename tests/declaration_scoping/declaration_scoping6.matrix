 {
	int q = 2;
	print q;
	{ 
		print q;
		int q = 22;
		print q;
		{
			print q;
			matrix q(2,2);
			print q;
		}
		print q;
	}
	print q;
 }
 