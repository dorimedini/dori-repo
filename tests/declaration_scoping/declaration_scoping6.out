2
2
22
22
[0,0;0,0]
22
2
